packages/account/src/hooks/__test__/useKycAuthStatus.spec.tsx

```diff
@@ -10,24 +10,44 @@ jest.mock('@deriv/api', () => ({
 }));
 
 describe('useKycAuthStatus', () => {
+    const mockRefetch = jest.fn();
```

Comment:
let's remove import React here as well

Reply:
Testcases need React import

Reply:
oh sorry.. this is not account package..
thanks @likhith-deriv